util.sh

```diff
@@ -47,6 +47,28 @@ function build_plist() {
     fi
 }
 
+################################################################################
+# Build font instances from designspace and UFOs.
+# Arguments:
+#     Path to designspace source.
+#     Output formats, as separate strings.
+################################################################################
+function build_designspace() {
+    filename=$(basename "$1"); filename="${filename%.*}"
+    fontmake -m "$1" -o "${@:2}" --expand-features-to-instances
```

Comment:
you need to add the `--interpolate` (or `-i`) option to produce "instances"; without it you only built the masters. Hence why they get generated in master_otf or master_ttf directories. You don't need them extra `cp` command below if you add the -i option.

```suggestion
    fontmake -m "$1" -i -o "${@:2}" --expand-features-to-instances
```